pep-0683.rst

```diff
@@ -23,7 +23,7 @@ Python's scalability.
 
 This proposal mandates that, internally, CPython will support marking
 an object as one for which that runtime state will no longer change.
-Consequently, such an object's refcount will never reach 0, and so the
+Consequently, such an object's refcount will never reach 0, and thus the
 object will never be cleaned up.  We call these objects "immortal".
```

Comment:
I'd like to add here that while these objects will never be cleaned up during the runtime, except, when the runtime knows it's safe to do safe. For example, at runtime shutdown when we clean and deallocate all strings.

Option 1:
and thus the object will never be cleaned up, unless explicitly deallocated by the runtime.

Option 2:
and thus the object will never be cleaned up, unless explicitly deallocated by the runtime when it knows it's safe to do so.

Reply:
Sounds good.

Reply:
@ericsnowcurrently Peanut gallery here, but it sounds like @eduardo-elizondo might have also wanted to know your preference between the above two options?